setup.py

```diff
@@ -14,7 +15,10 @@ class Config:
     CUDA_MIN_VER_MINOR = 7
     CUDA_MIN_VER_PATCH = 0
     CMAKE_GENERATOR = "'Unix Makefiles'"
-    VCPKG_INSTALL_DIR = os.path.join("external", "vcpkg")
+    RADAR_MSGS_DIR = os.path.join("external", "radar_msgs")
+    RADAR_MSGS_INSTALL_DIR = os.path.join("external", "radar_msgs", "install")
+    RADAR_MSGS_COMMIT = "47d2f26906ef38fa15ada352aea6b5aad547781d"
+    VCPKG_DIR = os.path.join("external", "vcpkg")
```

Comment:
Renamed to avoid semantic confusion other install dirs. *_DIR refers to location where package is downloaded, *_INSTALL_DIR refers to location where packages has been built, if relevant (install/ dir for ros2 packages)